### PR TITLE
Add Order class to abstract formatting into [price, amount, timestamp]

### PR DESCRIPTION
**Pull request by tmlee**
- What is the purpose of this Pull Request?
When implementing OrderBook, there is no structure for individual Order.
Here is an abstraction of a Class, when assigned with price, amount, and timestamp will spit out the format we expect for all exchanges

### CODE REVIEW

File: lib/cryptoexchange/models/order.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    class Order
+      attr_accessor :price, :amount, :timestamp
+
+      def initialize(args={})
+        @price      = args[:price]
+        @amount     = args[:amount]
+        @timestamp  = args[:timestamp] || nil
+      end
+
+      def format_as_list
```
Comment (dominicwong617) [inline lib/cryptoexchange/models/order.rb]: @tmlee Actually, why do we need to do this? Can't we just return an array of `Cryptoexchange::Models::Order` ?

Comment (tmlee) [reply lib/cryptoexchange/models/order.rb]: @dominicwong617 That is possible. We just need to decide what end result we want to provide. Though i think returning as `Order` would give the user best flexibility how they want to store the results.

Comment (tmlee) [reply lib/cryptoexchange/models/order.rb]: @dominicwong617 now giving the user the choice to call this if they want to

Comment (dominicwong617) [reply lib/cryptoexchange/models/order.rb]: @tmlee let's just return an array of object as asks and bids, this choice doesn't add much benefits

**Pull request merged**